ui/pages/confirm-transaction-base/low-gas-warning/low-gas-warning.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import ActionableMessage from '../../../components/ui/actionable-message/actionable-message';
+import { useGasFeeContext } from '../../../contexts/gasFee';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+const LowGasWarning = () => {
+  const { estimateToUse } = useGasFeeContext();
+  const t = useI18nContext();
+
+  if (estimateToUse !== 'low') return null;
+  return (
+    <div className="low-gas-warning">
+      <ActionableMessage
+        className="actionable-message--warning"
+        message={t('lowGasWarning')}
+        useIcon
+        iconFillColor="#f8c000"
```

Comment:
I *think* `COLORS.ALERT3` might work here.

Reply:
Good point, I fixed it.

Reply:
I found that `<ActionableMessage />` does not support use of these variables thus I reverted this change.

Reply:
[Q]: A little curious about if we can call `<ActionableMessage />` directly in `confirm-transaction-base.component` instead of creating a new component?

Reply:
`confirm-transaction-base.component` uses redux compose and accessing context was not straightforward thus I preferred child component like this.